main/zoe/api/zoe.md

```diff
@@ -183,24 +183,36 @@ const {
 } = await E(zoe).getOffer(offerHandle);
 ```
 
-## E(zoe).getCurrentAllocation(offerHandle, sparseKeywords)
+## E(zoe).getCurrentAllocation(offerHandle, brandKeywordRecord)
 - `offerHandle` <router-link to="/glossary/#handle">`{Array <Handle>}`</router-link>
-- `sparseKeywords` sparseKeywords is an array of string keywords, which may be a subset of allKeywords.
+- `brandKeywordRecord` <router-link to="/zoe/api/records.html#brand-keyword-record">`{[<BrandKeywordRecord>]}`</router-link>
+brandKeywordRecord is an optional record mapping keywords to brands for each of the offers.
 - Returns: <router-link to="/zoe/api/records.html#offer-record">`{<AmountKeywordRecord>}`</router-link>
 
-Get the amounts associated with the sparseKeywords for the offer.
+Get the amounts associated with the keywords for the offer. If brandKeywordRecord is not
```

Comment:
Let me try to clarify the phrasing;

For the offer's keywords, get their associated amounts. If `brandKeywordRecord` is not specified
only keywords with currently allocated amounts (including empty amounts) have their amounts returned.
If amounts to be included in the payout are not specified, they may be omitted.